plugins/modules/ecs_service.py

```diff
@@ -148,6 +148,14 @@
             description: A cluster query language expression to apply to the constraint.
             required: false
             type: str
+    purge_placement_constraints:
+        version_added: 5.3.0
+        description:
+            - Toggle overwriting of existing placement constraints. This is needed for backwards compatibility.
+            - By default I(purge_placement_constraints=false). In a release after 2024-06-01 this will be changed to I(purge_placement_constraints=true).
```

Comment:
```suggestion
            - By default I(purge_placement_constraints=false). In a release after 2024-06-01 this will be changed to I(purge_placement_constraints=true).
```

There should probably be a separate deprecation notice in the changelog too